nautobot/docs/installation/install_system.md

```diff
@@ -0,0 +1,475 @@
+    ```
+
+=== "RHEL8 + Derivatives"
+
+    ```no-highlight
+    sudo dnf check-update
+    sudo dnf install -y git python38 python38-devel python38-pip redis
+    ```
+
+## Database Setup
+
+In this step you'll set up your database server, create a database and database user for use by Nautobot, and verify your connection to the database.
+
+You must select either MySQL or PostgreSQL. PostgreSQL is used by default with Nautobot, so if you just want to get started or don't have a preference, please stick with PostgreSQL.
+
+Please follow the steps for your selected database backend below.
+
+=== "Ubuntu/Debian PostgreSQL"
+
+    ### Install PostgreSQL
+
+    This will install the PostgreSQL database server and client.
+
+    ```no-highlight
+    sudo apt install -y postgresql
+    ```
+
+    ### Create a PostgreSQL Database
+
+    At a minimum, we need to create a database for Nautobot and assign it a username and password for authentication. This
+    is done with the following commands.
+
+    !!! danger
+        **Do not use the password from the example.** Choose a strong, random password to ensure secure database authentication for your Nautobot installation.
+
+    ```no-highlight
+    sudo -u postgres psql
+    ```
+
+    Example output:
+
+    ```no-highlight
+    psql (12.5 (Ubuntu 12.5-0ubuntu0.20.04.1))
+    Type "help" for help.
+
+    postgres=# CREATE DATABASE nautobot;
+    CREATE DATABASE
+    postgres=# CREATE USER nautobot WITH PASSWORD 'insecure_password';
+    CREATE ROLE
+    postgres=# GRANT ALL PRIVILEGES ON DATABASE nautobot TO nautobot;
+    GRANT
+    postgres=# \q
+    ```
+
+    #### Verify PostgreSQL Service Status
+
+    You can verify that authentication works issuing the following command and providing the configured password. (Replace `localhost` with your database server if using a remote database.)
+
+    If successful, you will enter a `nautobot` prompt. Type `\conninfo` to confirm your connection, or type `\q` to exit.
+
+    ```no-highlight
+    psql --username nautobot --password --host localhost nautobot
+    ```
+
+    Example output:
+
+    ```no-highlight
+    Password for user nautobot:
+    psql (12.5 (Ubuntu 12.5-0ubuntu0.20.04.1))
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+    Type "help" for help.
+
+    nautobot=> \conninfo
+    You are connected to database "nautobot" as user "nautobot" on host "localhost" (address "127.0.0.1") at port "5432".
+    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, bits: 256, compression: off)
+    nautobot=> \q
+    ```
+
+=== "Ubuntu/Debian MySQL"
+
+    ### Install MySQL
+
+    This will install the MySQL database server and client. Additionally, MySQL requires that the MySQL development libraries are installed so that we may compile the Python `mysqlclient` library during the Nautobot installation steps.
+
+    ```no-highlight
+    sudo apt install -y libmysqlclient-dev mysql-server
```

Comment:
Note that in the Docker build we install `libmariadb-dev` rather than `libmysqlclient-dev` because the latter is GPL licensed. It might be good to add a note about this - again, non-blocking.